### DISCUSSION TITLE
Error: 'PrismaClient' is not exported by node_modules/@prisma/client/index.js

### DISCUSSION THREAD

**Discussion by Lynne42** [prisma/prisma#15140] [Q&A] [answered]
### Bug description

 Error: 'PrismaClient' is not exported by node_modules/@prisma/client/index.js, imported by src/helpers/client.ts

![image](https://user-images.githubusercontent.com/46595240/187872272-b5cbf926-cd17-47a5-b1f6-609cb97f6b65.png)

### How to reproduce

1. use
```
import {PrismaClient} from '@prisma/client';
let prisma = new PrismaClient();
export default prisma;
```

2. rollup.config.js
```
import ts from 'rollup-plugin-ts';
import commonjs from 'rollup-plugin-commonjs';
import {nodeResolve} from '@rollup/plugin-node-resolve';

export default {
    // 核心选项
    input: './index.ts',

    plugins: [
        ts({
            tsconfig: 'tsconfig.json',
        }),
        nodeResolve(),
        commonjs({
            include: 'node_modules/**',
        }),
    ],

    output: {
        // 核心选项
        file: './dist/index.js',
        format: 'cjs',
    },
};

```

3. build
```
rollup -c
```

### Prisma information

```
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model Session {
  id        Int      @id @default(autoincrement())
  name      String
  createdAt DateTime @default(now())
}
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```
prisma                  : 4.3.0
@prisma/client          : 4.3.0
Current platform        : darwin
Query Engine (Node-API) : libquery-engine c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/libquery_engine-darwin.dylib.node)
Migration Engine        : migration-engine-cli c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine    : introspection-core c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary           : prisma-fmt c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/prisma-fmt-darwin)
Format Wasm             : @prisma/prisma-fmt-wasm 4.3.0-32.c875e43600dfe042452e0b868f7a48b817b9640b
Default Engines Hash    : c875e43600dfe042452e0b868f7a48b817b9640b
Studio                  : 0.473.0

```

**Comment by austincrim** [comment, marked as answer]
Hey @qlniu 👋,

I was able to confirm this issue locally as well. Would you mind creating a minimal reproduction repository and then filing a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml)?

In the meantime, you can import directly from `node_modules/.prisma/client` as a workaround.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
